src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpExtractMethod.cs

```diff
@@ -131,7 +131,7 @@ public class Program
     public int Method()
     {
         Console.WriteLine(""Hello World"");
-        var result = [|NewMethod|]();
+        int result = [|NewMethod|]();
```

Comment:
?

Reply:
This is the default value for this code style option.

Reply:
ah. the editconfig root thing. +1